pkgs/development/compilers/dotnet/update.sh

```diff
@@ -51,7 +51,7 @@ platform_sources () {
     echo "    };"
 }
 
-sdk_packages () {
+aspnetcore_packages () {
```

Comment:
Any particular reason this is done in a seperate function? Seeing how they're all propagated for the SDK package unconditionally a comment in the `pkgs` list would suffice id say

Reply:
This is because the version for ASP.NET Core is different from the .NET Runtime version